libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -33,18 +33,19 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @Description: What action should be taken when fence is breached
     // @Values{Copter}: 0:Report Only,1:RTL or Land,2:Always Land,3:SmartRTL or RTL or Land,4:Brake or Land
     // @Values{Rover}: 0:Report Only,1:RTL,2:Hold,3:SmartRTL,4:SmartRTL or Hold
+    // @Values{Plane}: 0:Report Only,1:RTL,5:Guided,6:GuidedThrottlePass
     // @Values: 0:Report Only,1:RTL or Land
     // @User: Standard
     AP_GROUPINFO("ACTION",      2,  AC_Fence,   _action,        AC_FENCE_ACTION_RTL_AND_LAND),
 
-    // @Param{Copter, Sub}: ALT_MAX
+    // @Param: ALT_MAX
```

Comment:
Rover also probably will not want this

Reply:
Is there a use case where Rover may be climbing a mountain-side and exceed an altitude that needs to RTL?
I do note that it was not in the `AP_GROUPINFO_FRAME` definition, but, safest to check.